plans/integration/inhibit-if-kmods-is-not-supported/vm_centos7_bad.fmf

```diff
@@ -0,0 +1,8 @@
+discover+:
+    filter+:
+        - 'tag: centos7'
+        - 'tag: bad_tests'
+provision:
+    how: libvirt
+    origin_vm_name: c2r_centos7_template
+    develop: true
```

Comment:
Here's a hint how to define all three centos7 plan in a single file:

```yaml
discover+:
    filter+:
        - 'tag: centos7'

provision:
    how: libvirt
    origin_vm_name: c2r_centos7_template
    develop: true

/good:
    discover+:
        filter+:
          - 'tag: good_tests'

/bad:
    discover+:
        filter+:
          - 'tag: bad_tests'

/skip_check:
    discover+:
        filter+:
          - 'tag: skip_check'
```

Reply:
thank you Petr

Reply:
I did the same for tests